factcast-site/documentation-docsy/content/en/Setup/metrics.md

```diff
@@ -40,70 +40,68 @@ Furthermore, metrics in operations are automatically tagged with
 
-There are a bunch of metrics already emitted in the server. There are different kinds of metrics used:
+There are a bunch of metrics already emitted in the server. These metrics can be grouped by type:
 
 * Timers (collecting durations of code execution)
 * Meters (collecting metric events, for example, occurrences of errors)
 
 As this list is continuously growing, we cannot guarantee
 the documentation's completeness. If you want to see the current list of operations, please look
-at for instance [StoreMetrics.java](https://github.com/factcast/factcast/blob/master/factcast-store/src/main/java/org/factcast/store/internal/StoreMetrics.java)
+at [StoreMetrics.java](https://github.com/factcast/factcast/blob/master/factcast-store/src/main/java/org/factcast/store/internal/StoreMetrics.java)
 , [RegistryMetrics.java](https://github.com/factcast/factcast/blob/master/factcast-store/src/main/java/org/factcast/store/registry/metrics/RegistryMetrics.java)
 , or [ServerMetrics.java](https://github.com/factcast/factcast/blob/master/factcast-server-grpc/src/main/java/org/factcast/server/grpc/metrics/ServerMetrics.java) respectively.
 
 
 
-At the **time of writing (0.3.10)**, the store operations that are counted/measured are:
-
-| operation | duration  |
-|---|---|
-|    publish |  x |
-|    subscribe-follow |x |
-|    subscribe-catchup | x |
-|    fetchById | x |
-|    serialOf |  x |
-|    enumerateNamespaces | x |
-|    enumerateTypes |  x |
-|    getStateFor |  x |
-|    publishIfUnchanged | x |
-|    getSnapshot | x |
-|    setSnapshot  | x |
-|    clearSnapshot  | x |
-|    compactSnapshotCache  | x |
-|    notifyDatabaseRoundTrip | x |
-|    missedDatabaseRoundtrip | x |  
-
-At the **time of writing (0.3.10)**, the registry operations that are counted/measured are:
-
-| operation |  duration  |
-|---|---|
-| refreshRegistry | x |
-| compactTransformationCache | x |
-| transformEvent | x  |
-| tchRegistryFile | x |
-
-At the **time of writing (0.3.10)**, the registry events that are counted are:
-
-| event | meter  |
-|---|---|
-|    transformationCache-hit  | x | 
-|    transformationCache-miss | x | 
-|    missingTransformationInformation | x | 
-|    transformationConflict | x | 
-|    registryFileFetchFailed | x | 
-|    schemaRegistryUnavailable | x | 
-|    transformationFailed | x | 
-|    schemaConflict | x | 
-|    factValidationFailed | x | 
-|    schemaMissing | x | 
-|    schemaUpdateFailure | x |
-
-At the **time of writing (0.4.4)**, the server operations that are counted/measured are:
-
-| operation               | duration  |
-|-------------------------|---|
-| handshake               |  x |
-
+At the **time of writing (0.4.3)**, the metrics exposed by the namespaces group `factcast.server` are:
+
+| operation               | type    | description                      |
+|-------------------------|---------|----------------------------------|
+| handshake               | `timer` | Duration of the GRPC handshakes. |
```

Comment:
s/GRPC/initial 

This is slightly less than the GRPC handshake, because it only measures the application level call, excluding marshalling and GRPC mechanics.